tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,31 @@
+""" Handles logging functionality. """
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority,publisher_id,**log_payload):
+    """
+    Function used for logging errors/output/info.
+    Args:
+        log_priority : Priority of the Log Message (error/info)
+        publisher_id : Id of publisher (mandatory)
+        log_payload  : Payload can contain parameters like message that is to be logged
+    """
+    try:
+        if publisher_id != None:
+            Event(
+                Message(
+                    priority=log_priority,
+                    publisher=publisher_id,
+                    payload=log_payload
+                )
+            )
+        else:
+            raise Exception('Publisher Id not found')
+    except:
+            message = log_payload.get("message","Meesage Not Found") or log_payload.get("Message","Meesage Not Found")
```

Comment:
I think rather than using log_payload.get,  log_payload["message"] should be used as each log should have a massage. If it is not there then that might be an error, as then there will be no point of logging. Also I think only lower case should be checked as everywhere it is lower. If you find capital then you can change it.